packages/fields-cloudinary-image/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@keystonejs/fields-cloudinary-image",
+  "description": "KeystoneJS Cloudinary Image Field Type",
+  "version": "1.0.0",
```

Comment:
This should start at `0.0.0`, changesets will take of the initial bump to `1.0.0`.

Reply:
Also, the changesets should contain a `major` bump with an initial commit message for each of the new packages.